packages/react-core/src/demos/DataListDemo.md

```diff
@@ -123,7 +123,7 @@ class ExpandableDataList extends React.Component {
                     variant="plain"
                     aria-label={this.state.allExpanded ? 'Collapse all rows' : 'Expand all rows'}
                   >
-                    <ToolbarExpandIconWrapper>
+                    <ToolbarExpandIconWrapper className="pf-v5-m-mirror-inline-rtl">
```

Comment:
You could be able to update this to use the prop now that your other PR went in

Reply:
That would just require using our Icon component here as well. Since this is a demo may not be a big deal so I can do that if that sounds good to you